CodeConverter/Util/FromRoslyn/ITypeSymbolExtensions.cs

```diff
@@ -254,6 +254,36 @@ public static bool IsUnexpressibleTypeParameterConstraint(this ITypeSymbol typeS
             return false;
         }
 
+        public static bool IsFloatingNumericType(this ITypeSymbol? type)
```

Comment:
Perhaps a better name would be something like `IsFractionalNumericType` - since a decimal doesn't have a floating point this implementation surprised me